app-frontend/src/app/components/scenes/sceneItem/sceneItem.controller.js

```diff
@@ -42,4 +42,9 @@ export default class SceneItemController {
             }
         }
     }
+
+    getReferenceDate() {
+        let aqDate = this.scene.filterFields.acquisitionDate;
```

Comment:
`acDate` or `acqDate` perhaps? `aqDate` strikes me as an odd way to abbreviate "acquisition".